test/pyscriptTests.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
Should this make sure that python=python3?

Reply:
I don't think it's all that important (the script will most likely just fail on Python 2) but I can change it to use `python3` in the shebang.

Reply:
Done.

Reply:
Nowadays they're the same on most systems I used plain `python` mostly because I was wary of a situation where `python` is available (and is Python 3) but `python3` is not. I haven't seen such a situation in practice though so, yeah, maybe let's use `python3` consistently.